src/reference/build.gradle

```diff
@@ -0,0 +1,72 @@
+buildscript {
+  dependencies {
+    classpath 'org.asciidoctor:asciidoctorj-epub3:1.5.0-alpha.6'
+    classpath 'org.asciidoctor:asciidoctorj-pdf:1.5.0-alpha.15'
+  }
+}
+
+plugins {
+  id 'de.undercouch.download' version '3.4.3'
+  id 'org.asciidoctor.convert' version '1.5.3'
+}
+
+apply plugin: 'java'
+apply plugin: 'org.asciidoctor.convert'
+
+version = '1.0.0-SNAPSHOT'
```

Comment:
What is this version about?

Reply:
No clue. I inherited that from the script I borrowed to get started. I think it can be removed. I'll have to test.